docs/reference-guides/block-api/block-selectors.md

```diff
@@ -1,5 +1,12 @@
 # Selectors
 
+<div class="callout callout-alert">
+	This API was stabilized in Gutenberg 15.5 and is planned for core release
+	in WordPress 6.3. To use this prior to WordPress 6.3, you will need to
+	install and activate Gutenberg >= 15.5.
+</div>
+<br />
```

Comment:
I don't see a line break added in other places, but we can verify once the handbook updates.